models/safe/zksync/safe_zksync_transactions.sql

```diff
@@ -0,0 +1,112 @@
+{{ 
+    config(
+        materialized='incremental',
+        
```

Comment:
can you add the `schema` property in the config?

right now, it's likely picking up from dbt_project file, but we want to move away from that approach moving forward.

Reply:
Done.